mscore/uploadscoredialog.cpp

```diff
@@ -233,6 +233,10 @@ void UploadScoreDialog::onGetScoreSuccess(const QString &t, const QString &desc,
       // file with score info
       title->setText(t);
       description->setPlainText(desc);
+      QScreen* screen      = QGuiApplication::primaryScreen();
```

Comment:
Unfortunately I don't know what the strategy *should* be, I just know some things that *don't* tend to work well.